pom.xml

```diff
@@ -59,7 +59,7 @@ limitations under the License.
         <hamcrest.version>1.3</hamcrest.version>
 
         <!-- google-cloud-java related dependency versions -->
-        <google-cloud.version>0.60.0-alpha</google-cloud.version>
+        <google-cloud.version>0.62.0-alpha</google-cloud.version>
```

Comment:
The version bump should happen outside of this PR

Reply:
Yes, I will create different PR for version upgrade once we have unsafe mutation available in google cloud java.